### PR TITLE
Formatting and formula fixes

### PR DESCRIPTION
**Pull request by andrewwhitehead**
This PR applies changes to the markdown to produce a more legible HTML document, as well as a couple fixes, annotated in comments.

### CODE REVIEW

File: spec.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ keyword = [""]
 
 [seriesInfo]
 name = "Individual-Draft"
-value = "bbs-signatures"
+value = "bbs-signatures-00"
```
Comment (andrewwhitehead) [inline spec.md:12]: Generator won't run without a version number

File: spec.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ organization = "Mattr"
   email = "tobias.looker@mattr.global"
 %%%
 
-# Abstract
+.# Abstract
```
Comment (andrewwhitehead) [inline spec.md:34]: Fixes HTML generation

File: spec.md [hunk 3]
```diff
@@ -95,7 +95,7 @@ D
 msg
 : The input to be signed by the signature scheme.
    
-h\_i
+h\[i\]
```
Comment (andrewwhitehead) [inline spec.md:98]: Applied globally for consistency, as subscripts are not rendered

File: spec.md [hunk 4]
```diff
@@ -229,15 +229,19 @@ SK = KeyGen(IKM)
 ```
 
 Inputs:
+
```
Comment (andrewwhitehead) [inline spec.md:232]: (And related) Fix formatting to use an unordered list

File: spec.md [hunk 5]
```diff
@@ -285,25 +291,27 @@ The SkToPk algorithm takes a secret key SK and the number of messages that can b
 SK MUST be indistinguishable from uniformly random modulo r (Section 2.2) and infeasible to guess, e.g., generated using a trusted source of randomness.  KeyGen (Section 2.3) outputs SK meeting these requirements.  Other key generation approaches meeting these requirements MAY also be used; details of such methods are beyond the scope of this document.
 
 ```
-PK = SkToPk(Sk, count)
+PK = SkToPk(Sk, L)
```
Comment (andrewwhitehead) [inline spec.md:294]: Switched count to L globally for consistency

File: spec.md [hunk 6]
```diff
@@ -422,9 +438,9 @@ Procedure:
 
 4. if KeyValidate(pub\_key) is INVALID
 
-5. b = P1 + h0 \* s + h\_i \* msg\_i + ... + h\_n \* msg\_n
+5. b = P1 + h0 \* s + h\[i\] \* msg\[i\] + ... + h\[n\] \* msg\[L\]
 
-6. C1 = e(A, w \* P2 ^ e)
+6. C1 = e(A, w + P2 \* e)
```
Comment (andrewwhitehead) [inline spec.md:443]: Fixed formula

File: spec.md [hunk 7]
```diff
@@ -693,15 +723,15 @@ Procedure:
 
 7. if X1 != X2 return INVALID
 
-8. c = H(Abar || A' || h0 || C1 || d || h0 || h\_i || ... || h\_R || C2 || nonce)
+8. c = H(Abar || A' || h0 || C1 || d || h0 || h\[i\] || ... || h\[R\] || C2 || nonce)
 
 9. T1 = Abar - d
 
-10. T2 = P1 + h\_i \* -m^\_i + ... + h\_D \* -m^\_D
+10. T2 = P1 + h\[i\] \* msg\[i\] + ... + h\[D\] \* msg\[D\]
```
Comment (andrewwhitehead) [inline spec.md:730]: Updated the formula here, and in item 12. The previous version added `P1*c` to `Y2` instead of subtracting it.

**Review by tplooker — APPROVED** [review]

**Comment by tplooker** [comment]
LGTM thanks @andrewwhitehead

**Pull request merged**